node/src/isolate/external.cpp

```diff
@@ -232,30 +252,12 @@ struct external_t::inner_t :
         });
     }
 
-    void reconnect(const std::error_code& e) {
-        if(!connecting) {
-            connecting = true;
-            if(session) {
-                session->detach(e);
-                session.reset();
-            }
-
-            auto reconnect_timeout_ms = args.as_object().at("reconnect_timeout_ms", 3000u).as_uint();
-            COCAINE_LOG_INFO(log, "queueing reconnect to external isolation daemon after {} ms", reconnect_timeout_ms);
-
-            connect_timer.expires_from_now(boost::posix_time::milliseconds(reconnect_timeout_ms));
-            std::weak_ptr<inner_t> weak_self(shared_from_this());
-            connect_timer.async_wait([=](const std::error_code&) {
-                // We don't perform error code check as nobody should cancel this timer
-                auto self = weak_self.lock();
-                if (self) {
-                    self->connect();
-                } // else application was already stopped and isolation was destroyed
-            });
-        }
-    }
-
     void on_fail(const std::error_code& ec) {
+        last_failed_connect_time = std::chrono::system_clock::now();
+        seal_time *= 2;
```

Comment:
This can be rewritten as `seal_time = std::min(seal_time * 2, max_seal_time)` but whatever.